vnext/JSI/Desktop/JSI.Desktop.vcxproj

```diff
@@ -29,9 +29,6 @@
   <PropertyGroup Label="Globals">
     <ProjectGuid>{17DD1B17-3094-40DD-9373-AC2497932ECA}</ProjectGuid>
     <ProjectName>JSI.Desktop</ProjectName>
-    <!-- Disabled until we reconcile JSI interface versions -->
-    <USE_V8>false</USE_V8>
-    <V8AppPlatform>win32</V8AppPlatform>
```

Comment:
Did you mean to remove the `V8AppPlatform` here?

Reply:
Yes, it turns out V8 doesn't need to be referenced in this project at all.

---
In reply to: [483757818](https://github.com/microsoft/react-native-windows/pull/5918#discussion_r483757818) [](ancestors = 483757818)